build/webpack.config.js

```diff
@@ -127,86 +128,32 @@ webpackConfig.module.loaders = [{
 // ------------------------------------
 // Style Loaders
 // ------------------------------------
-// We use cssnano with the postcss loader, so we tell
-// css-loader not to duplicate minimization.
-const BASE_CSS_LOADER = 'css?-minimize'
-
-// Add any packge names here whose styles need to be treated as CSS modules.
-// These paths will be combined into a single regex.
-const PATHS_TO_TREAT_AS_CSS_MODULES = [
-  // 'react-toolbox', (example)
-]
-
-// If config has CSS modules enabled, treat this project's styles as CSS modules.
-if (config.compiler_css_modules) {
-  PATHS_TO_TREAT_AS_CSS_MODULES.push(
-    paths.universal().replace(/[\^\$\.\*\+\-\?\=\!\:\|\\\/\(\)\[\]\{\}\,]/g, '\\$&') // eslint-disable-line
-  )
-}
-
-const isUsingCSSModules = !!PATHS_TO_TREAT_AS_CSS_MODULES.length
-const cssModulesRegex = new RegExp(`(${PATHS_TO_TREAT_AS_CSS_MODULES.join('|')})`)
-
-// Loaders for styles that need to be treated as CSS modules.
-if (isUsingCSSModules) {
-  const cssModulesLoader = [
-    BASE_CSS_LOADER,
-    'modules',
-    'importLoaders=1',
-    'localIdentName=[name]__[local]___[hash:base64:5]'
-  ].join('&')
-
+if (__DEV__) {
   webpackConfig.module.loaders.push({
-    test: /\.scss$/,
-    include: cssModulesRegex,
+    test: /\.css$/,
```

Comment:
@samcdavid it can do both simultaneously just fine from what i've read in the stylelint docs. I'll give it a go sometime soon and we'll see! 

I like the shorter syntax with SugarSS, but i freaking hate the nesting and referring globals...and generally making it much more difficult than it needs to be to read. If we can use it but also enforce no globals on the component level, I might be ok with it.

IMHO there is a lot of talk of using SASS/SCSS, globals, and generally regressing to using css pre-processing. I'd really love for us to fight against that unless someone can give explicit reasons why we should use the more complicated approach vs. post-css and all the nice stuff it provides out of the box (cssNext, autoprefixing, etc.) 

ref: @Mrjaco12 interested in your thoughts too...I really don't want to go a certain direction just because it's what the designers are "comfortable" with.